src/components/selectboxes.jsx

```diff
@@ -0,0 +1,50 @@
+
+var React = require('react');
+var valueMixin = require('./mixins/valueMixin.jsx');
+
+module.exports = React.createClass({
+  displayName: 'SelectBox',
+  mixins: [valueMixin],
+  componentWillMount: function() {
```

Comment:
I don't think this is the right way of setting the initial value. First we shouldn't set the state variables directly as it bypasses update functionality based on the changing data and second it won't work with existing data being passed in.

We already have a function for setting the initial state called getInitialValue at https://github.com/formio/react-formio/blob/master/src/components/mixins/valueMixin.jsx#L8. You can use this to set the default value for the component.